local-volume/provisioner/pkg/deleter/deleter.go

```diff
@@ -140,6 +145,12 @@ func (d *Deleter) asyncDeletePV(pv *v1.PersistentVolume, volMode v1.PersistentVo
 
 	glog.Infof("Deleted PV %q", pv.Name)
 
+	action := metrics.ProvisionDelete
+	metrics.PersistentVolumeProvisionTotal.WithLabelValues(action).Inc()
+	procEntry := d.ProcTable.GetProcEntry(pv.Name)
+	if procEntry != nil {
+		metrics.PersistentVolumeProvisionLatency.WithLabelValues(action).Observe(time.Since(procEntry.StartTime).Seconds())
```

Comment:
For delete, would it be useful to also break it out by PV capacity and delete method?

Reply:
It should be useful.

VolumeMode determines delete method, we can use VolumeMode here like `PersistentVolumeCleanupFailedTotal`.

Values of label in metrics can not have high cardinality, but we can classify capacity into:

- 1M
- 10M
- 100M
- 1000M
- 10000M
- 100000M
- ...

what do you think?

Reply:
Maybe breaking it down into every 500gb could work. We mostly care about larger capacities where cleanup can take a very long time.

For delete action, we actually have customizable cleanup routines, some of which can be very fast (wipe fs), and some can be very slow (shred).  So that may be good to distinguish in the metrics.  Or May it's good enough to categorize by storageclass? A single storageclass can only have one cleanup method

Reply:
Understood. I will use cleanup command here. Using storageclass is not good, because storageclass can be deleted from apiserver or recreated with different configuration.